Content.Shared/Containers/ThrowInsertContainerComponent.cs

```diff
@@ -1,22 +1,22 @@
 using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
 
-namespace Content.Server.Containers;
+namespace Content.Shared.Containers;
 
 /// <summary>
 /// Allows objects to fall inside the Container when thrown
 /// </summary>
-[RegisterComponent]
-[Access(typeof(ThrowInsertContainerSystem))]
+[RegisterComponent, NetworkedComponent]
 public sealed partial class ThrowInsertContainerComponent : Component
 {
     [DataField(required: true)]
     public string ContainerId = string.Empty;
 
     /// <summary>
-    /// Throw chance of hitting into the container
+    /// Probability of missing the container
     /// </summary>
     [DataField]
-    public float Probability = 0.75f;
+    public float BaseHitProbability = 0.25f;
```

Comment:
I would see if it's possible to look at making this more of a 75% chance to hit as the intended code was actually in reverse and should have been a chance of 75% to get it inside the Bin.

I was also seeing if maybe you could just set this to be more like 0.9f so it's a 1/10 chance of getting it in the disposal bin. Make it more of a funny thing when it misses and base all your code of the higher chance to get in, making it severely worse when you have any form of eye damage or condition.